packages/react/src/ErrorBoundaryGroup.tsx

```diff
@@ -29,13 +29,13 @@ export const ErrorBoundaryGroup = ({
 }) => {
   const [resetKey, reset] = useKey()
   const parentGroup = useContext(ErrorBoundaryGroupContext)
-  const previousParentGroup = usePrevious(parentGroup)
+  const isChangedParentGroupResetKey = useIsChanged(parentGroup?.resetKey)
```

Comment:
why not using this variable name?!

```tsx
const `isParentGroupResetKeyChanged` = useIsChanged(parentGroup?.resetKey)
```

is...Changed is more appropriate I think! than isChanged....

Reply:
Could you use suggest code of github features. I want to commit your suggestion.
<img width="216" alt="image" src="https://github.com/suspensive/react/assets/61593290/83535cd2-7d86-4743-bf31-bfb0857225b4">

Reply:
I renamed `isChangedParentGroupResetKey` to `isParentGroupResetKeyChanged` and added you as co-author in [d9cd0a8](https://github.com/suspensive/react/pull/92/commits/d9cd0a8c44f0a5456919e19d49ec6b83bbf01b25)